wandb/sdk/wandb_run.py

```diff
@@ -3419,6 +3431,11 @@ def _footer_sync_info(
                 info = [
                     f"{printer.emoji('rocket')} View run {printer.name(settings.run_name)} at: {printer.link(settings.run_url)}"
                 ]
+            if job_info and job_info.version and job_info.sequenceId:
+                link = f"{settings.project_url}/jobs/{job_info.sequenceId}/version_details/{job_info.version}"
+                info.append(
+                    f"{printer.emoji('lightning')} View job at {printer.link(link)}",
+                )
```

Comment:
could you move into it's own utility method?

Reply:
Easy change to make, but I would actually prefer that we have the run link and the job link together in the output. Is that okay?

Reply:
just put the prints one after the other in the footer function?

Reply:
The thing is that this _footer_sync_info includes a line with output before and after the line with the run link, so I think I would need to break this function up even more or change the order that it is outputting the lines if I want to put the job link in its own helper while keeping it adjacent to the run link.

Reply:
yeah, so keep it like this... you could just re-order things, but that's fine.